ansible_rulebook/messages.py

```diff
@@ -16,5 +16,6 @@
 
 
 class Shutdown(NamedTuple):
-
-    pass
+    message: str = "Not specified"
```

Comment:
@benthomasson This is the formal definition of Shutdown which has all the fields, except type which is needed for event_log.